docs/ObjC.md

```diff
@@ -106,4 +134,49 @@ Objective-C surfaced API:
 @end
 ```
 
-Here we can see that `initWithId:` has been marked as unavailable.
\ No newline at end of file
+Here we can see that `initWithId:` has been marked as unavailable.
+
+### Operator
+
+ObjC does not support operator overloading as C# does, so operators are converted to class selectors:
+
+```
+	public static AllOperators operator + (AllOperators c1, AllOperators c2)
+	{
+		return new AllOperators (c1.Value + c2.Value);
+	}
+```
+
+to
+
+```
++ (instancetype)addition:(Overloads_AllOperators *)anObjectC1 c2:(Overloads_AllOperators *)anObjectC2;
+```
+
+However, some .NET languages do not support operator overloading so it is common to also include a ["friendly"](https://msdn.microsoft.com/en-us/library/ms229032(v=vs.110).aspx) named method in addition.
+
+If both the operator version and the "friendly" version are found, only the friendly version will be generated.
+
+```
+	public static AllOperatorsWithFriendly operator + (AllOperatorsWithFriendly c1, AllOperatorsWithFriendly c2)
+	{
+		return new AllOperatorsWithFriendly (c1.Value + c2.Value);
+	}
+
+	public static AllOperatorsWithFriendly Add (AllOperatorsWithFriendly c1, AllOperatorsWithFriendly c2)
+	{
+		return new AllOperatorsWithFriendly (c1.Value + c2.Value);
+	}
+```
+
+becomes:
+
+```
++ (instancetype)addC1:(Overloads_AllOperatorsWithFriendly *)anObjectC1 c2:(Overloads_AllOperatorsWithFriendly *)anObjectC2;
+```
+
+### Equality operator
+
+In general operator == in C# is handled as a general operators as noted above.
```

Comment:
In general operator == in C# is handled as a general operator as noted above.

(No plural on operator)